pavelib/utils/test/suites/bokchoy_suite.py

```diff
@@ -104,7 +104,10 @@ def __exit__(self, exc_type, exc_value, traceback):
         print msg
 
         # Clean up data we created in the databases
-        sh("./manage.py lms --settings bok_choy flush --traceback --noinput")
+        if not self.testsonly:
```

Comment:
Should clearing the mongo DB be inside the loop also? (I'm not sure, I guess I could go either way on this.)

Also now the "Cleaning up databases..." message prints out to the console either way, but what gets cleaned will be different depending on whether or not the testsonly flag was flown. Maybe instead of outputting that generic message in both cases, it would be better to have 2 discreet messages. Something like "Flushing the bok_choy database" and "Clearing the mongo database."